pdns/fstrm_logger.hh

```diff
@@ -39,15 +39,30 @@ public:
   FrameStreamLogger(int family, const std::string& address, bool connect, const std::unordered_map<string,unsigned>& options = std::unordered_map<string,unsigned>());
   ~FrameStreamLogger();
   [[nodiscard]] RemoteLoggerInterface::Result queueData(const std::string& data) override;
+
+  std::string address() const override
+  {
+    return d_address;
+  }
+
   const std::string name() const override
```

Comment:
Unrelated to this PR, but if you add the `[[nodiscard]]` above then also add it here, and remove the `const` (I guess they're going to have to change in the base class as well).